checkboxes/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdkVersion versions.compileSdk
+
+    defaultConfig {
+        minSdkVersion versions.minSdk
+        targetSdkVersion versions.compileSdk
+        consumerProguardFiles "consumer-rules.pro"
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+}
+
+dependencies {
+    api(project(":foundation"))
+    api(deps.material)
```

Comment:
Should not be necessary. `deps.material` dependency is already defined on `foundation` module with `api` call to be accessible for all module consumers.

Reply:
Updated ✅ 
I also change it in the button module